### DISCUSSION TITLE
We got a 401 code when we applied for an EAS request

### DISCUSSION THREAD

**Discussion by DoctorHe** [AzureAD/microsoft-authentication-library-for-android#1810] [Q&A]
My interest team is having some problems trying to adapt ADAL to MSAL.
I have an Outlook developer account. I made a series of settings according to the development instructions in the related documentation. I successfully obtained the access token and concatenated it into the bearer string. However, we returned code 401 when requesting the Exchange Active Sync service. I'd be grateful if you could answer my question.

**Comment by negoe** [comment]
Hi @DoctorHe Can you please create an issue [here ](https://github.com/AzureAD/microsoft-authentication-library-for-android/issues)and provide logs or code details if possible?

**Reply by DoctorHe** [reply]
Thanks,I have solved this problem.
